app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/NavigationOverlayFragment.kt

```diff
@@ -294,6 +271,17 @@ class NavigationOverlayFragment : Fragment() {
         ).map { channel -> channel.forwardRemoveEventsToRepo() }
     }
 
+    private fun observeToolbarState(): Disposable {
```

Comment:
Nit: could we rename this one to `observeToolbarFocusability`, or something similar?  To distinguish this from code that sets when buttons are enabled and such.